### ISSUE TITLE
Add support for higher vram GPUs (24GB+)

### ISSUE THREAD

**Issue by aperire** [cocktailpeanut/fluxgym#267]
Hi, I understand that fluxgym was designed to support low vram cards to train Loras. However, due to it's simple UI, there are a lot of users who have 4090 or ones waiting for 5090s (36GB vram) who are willing to use Fluxgym.

Support for higher vrams would be a great decision without sacrificing any convenience for other users.

**Comment by vivek-kumar-poddar** [comment]
Need this. Please add 24, 36, 48 or a field to enter the custom GPU memory size. Much needed when using platforms like Vultr or Runpod. Thanks!

**Comment by K-O-N-B** [comment]
yes, 4090 is waitting for it.

**Comment by chikiu-san** [comment]
Agree, love the simplistic design compared to other scripts. If we can have flexibility on specifying VRAM, that would be great.
